app/src/main/res/values-land/dimens.xml

```diff
@@ -390,4 +391,7 @@
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">72dp</dimen>
 
+  <!-- Story Fragment- land dimens -->
```

Comment:
Just name it as `<!-- StoryFragment -->`

Reply:
okk, I will change it.